BitwardenShared/UI/Autofill/Utilities/Fido2UserInterfaceHelper.swift

```diff
@@ -94,6 +94,9 @@ class DefaultFido2UserInterfaceHelper: Fido2UserInterfaceHelper {
     func pickCredentialForAuthentication(
         availableCredentials: [BitwardenSdk.CipherView]
     ) async throws -> BitwardenSdk.CipherViewWrapper {
+        if availableCredentials.count == 1 {
+            return CipherViewWrapper(cipher: availableCredentials[0])
```

Comment:
🤔 Is it worth writing a test for this right now, or better to wait for 8829 and do the whole thing in one fell swoop?